packages/components/src/internal/components/ontology/OntologyConceptPicker.tsx

```diff
@@ -47,6 +48,22 @@ export const OntologyConceptPicker: FC<Props> = memo((props: Props) => {
         }
     }, [conceptSubtree]);
 
+    const showPickerModal = useCallback(()=>{
+        setShowPicker(true);
+    },[showPickerListener])
```

Comment:
is `showPickerListener` the right dep here? I don't think you need any.

Reply:
Removed due to removal of footer link.